src/main/modules/draft-store/index.ts

```diff
@@ -0,0 +1,29 @@
+import config from 'config';
+import { Application } from 'express';
+import Logger from 'winston';
+
+const Redis = require('ioredis');
+
+export class DraftStoreClient {
+  public static REDIS_CONNECTION_SUCCESS = 'Connected to Redis instance successfully';
+
+  public enableFor(app: Application): void {
+    // const protocol = config.get('services.draftStore.redis.tls') ? 'rediss://' : 'redis://';
+    // const connectionString = `${protocol}${config.get('session.redis.key')}@${config.get(
```

Comment:
currently commented to make sure everything works on higher environments as well. Once tested on demo/aat this can be removed.